curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c8ad0cd8f1e5201c4ef2e4.md

```diff
@@ -7,34 +7,26 @@ dashedName: step-56
 
 # --description--
 
-The `list` parameter is going to be the result of a query selector, which will return a `NodeList`. A `NodeList` is an array-like which contains the elements that match the query selector. 
+The `list` parameter is going to be the result of a query selector, which will return a `NodeList`. A `NodeList` is an array-like which contains the elements that match the query selector. You will need to loop through these elements in the list. 
 
-You'll need to loop through these elements, so start by creating a `for` loop. Your iterator `i` should start at `0`, continue while it is less than the length of the list, and increment by `1` each iteration.
+In previous steps, you learned how to loop through an array using a `for` loop. However, you can also loop through a `NodeList` using a <dfn>for...of</dfn> loop.
```

Comment:
I think we should use backticks to highlist the words ? or maybe use both dfn and backticks ?
```suggestion
In previous steps, you learned how to loop through an array using a `for` loop. However, you can also loop through a `NodeList` using a `for...of` loop.
```

Reply:
For the first introduction of a concept we use the `<dfn></dfn>` then other mentions use the back ticks

Reply:
@jdwilkin4  i see ! my bad, i must've misread the guidelines